ra_aid/proc/interactive.py

```
@@ -1,90 +1,158 @@
+        old_settings = termios.tcgetattr(stdin_fd)
+        tty.setraw(stdin_fd)
+        try:
+            while True:
+                rlist, _, _ = select.select([master_fd, stdin_fd], [], [])
+                if master_fd in rlist:
+                    try:
+                        data = os.read(master_fd, 1024)
+                    except OSError as e:
+                        if e.errno == errno.EIO:
+                            break
+                        else:
```

Ruff (0.8.2)

Unnecessary `else` after `break` statement

Remove unnecessary `else`

(RET508)